tb_plugin/torch_tb_profiler/profiler/module_parser.py

```diff
@@ -156,3 +157,4 @@ def aggregate(self, tid2tree):
         self.op_list_groupby_name_input = list(agg_result[1].values())
         self.stack_lists_group_by_name = stack_lists_group_by_name
         self.stack_lists_group_by_name_input = stack_lists_group_by_name_input
+        self.tc_eligible_ops = sum(map(lambda op : op.tc_eligible, ops))
```

Comment:
seems list comprehension is faster than map if using lambda? https://stackoverflow.com/questions/1247486/list-comprehension-vs-map

Reply:
@guotuofeng Cool! What about "sum([1 if op.tc_eligible else 0 for op in ops])"?

Reply:
@guotuofeng Compared the speed, they are nearly same(<20% difference). I'll use "sum([1 if op.tc_eligible else 0 for op in ops])" for easy to understand.